test/zdtm.py

```diff
@@ -1961,8 +1961,9 @@ def run_test(self, name, desc, flavor):
         with open("/proc/sys/kernel/tainted") as taintfd:
             taint = taintfd.read()
         if self.__taint != taint:
-            raise Exception("The kernel is tainted: %r (%r)" %
-                            (taint, self.__taint))
+            if not opts["ignore_taint"]:
```

Comment:
Eh, it was intentional. The idea was to detect changes.

Reply:
I expected that it was intentional. Not sure how to handle the test failure in another way. As long as the Ubuntu kernel breaks with a CentOS 8 user space it will always report an error which might hide some other error.